### ISSUE TITLE
genkernel-next v69 (latest stable), dependency of buildkernel, is now masked because it does not build with recent glibc

### ISSUE THREAD

**Issue by LIII-XXII** [sakaki-/buildkernel#35]
Just a heads up: I am now getting this message upon updating my system with buildkernel installed:

```
!!! The following installed packages are masked:
- sys-kernel/genkernel-next-69::gentoo (masked by: package.mask)
/usr/portage/profiles/package.mask:
# Andreas K. Hüttel <dilfridge@gentoo.org> (2020-07-11)
# Fails to build with recent glibc, bug 719968
# Removal in 30 days
```

**Comment by sakaki-** [comment]
Thanks. As noted in https://github.com/sakaki-/sakaki-tools/issues/22, I recommend unmasking this pro tem (the guide has also been updated to reflect this); we'll see what [happens upstream](https://bugs.gentoo.org/719968) as it may get proxy maintained.

**Comment by lazoz** [comment]
Hello,

```
   The following unavailable installed packages were found
             sys-kernel/genkernel-next-69
```

eix-diff seems to indicate that sys-kernel/genkernel-next was removed in the last 24 hours. Unmasking it will not resolve the issue anymore.
`[?]   << sys-kernel/genkernel-next ({M}69{tbz2}): Gentoo automatic kernel building scripts, reloaded`

Rgds,

**Comment by reanimus** [comment]
Correct, the package has been removed. Is it possible to move to the regular genkernel package?

**Comment by KDCadet** [comment]
Sakaki, what can we do to help you update buildkernel to use the regular genkernel?

**Comment by sakaki-** [comment]
31 Oct 2020: sadly, due to legal obligations arising from a recent change in my 'real world' job, I must announce I am **standing down as maintainer of this project with immediate effect**. For the meantime, I will leave the repo up (for historical interest, and it may be of use to others); however, I plan no further updates, nor will I be accepting / actioning further pull requests or bug reports from this point. Email requests for support will also have to be politely declined, so, **please treat this as an effective EOL notice**.<br><br>For further details, please see my post [here](https://forums.gentoo.org/viewtopic-p-8522963.html#8522963).<br><br>With sincere apologies, sakaki ><
